Project/src/MakeCall/CallCard.js

```diff
@@ -562,18 +562,14 @@ export default class CallCard extends React.Component {
                     this.setState({ videoOn: true })
                 }
                 await this.watchForCallFinishConnecting();
-                if (this.state.videoOn) {
-                    if (this.state.canOnVideo) {
-                        await this.call.startVideo(this.localVideoStream);
-                    }
+                if (this.state.videoOn && this.state.canOnVideo) {
```

Comment:
A more proper use of this flag then would be to disable/enable the actual video button on the UI